test/types/scalar/bradc/valAsType.good

```diff
@@ -1,2 +1 @@
-valAsType.chpl:1: syntax error: near ';'
-valAsType.chpl:2: syntax error: near ';'
+valAsType.chpl:1: error: invalid type specification
```

Comment:
That's a nice improvement!

Reply:
I'd love to take credit for it, but it turns out someone else already accounted for the possibility of the grammar changing and wired up that check in the compiler proper 😄.

Reply:
I think if your change enabled their error to fire, you still get partial credit for getting it over the finish line!